src/Converter/ShipmentLineItemsConverter.php

```diff
@@ -50,14 +50,31 @@ private function convertUnitRefundToLineItem(ShipmentRefund $shipmentRefund): Li
         Assert::isInstanceOf($shipment, AdjustableInterface::class);
         Assert::lessThanEq($shipmentRefund->total(), $shipment->getAdjustmentsTotal());
 
+        $grossValue = $shipmentRefund->total();
+
+        $taxAdjustment = $shipment->getAdjustments(AdjustmentInterface::TAX_ADJUSTMENT)->first();
+        $taxRate = $this->getTaxRateAmount($taxAdjustment) . '%';
```

Comment:
Below you are multiplying the tax rate by 100(see attached image), but not here
![image](https://user-images.githubusercontent.com/6213903/110774169-cabebf80-825d-11eb-8ede-f14fda664f13.png)

Which solution is proper? And also, as this logic is so crucial, can we extract another service for it?

Reply:
I've extracted it to a new service. I multiply it when I need string with taxRate, I left it as a float when i need `taxRate amount`